.changeset/proud-laws-arrive.md

```diff
@@ -0,0 +1,68 @@
+---
+"@comet/admin": minor
+---
+
+Add `ChipSelect` component
+
+This adds the ChipSelect component, as well as a wrapper for FinalForm (`FinalFormChipSelect`) and an abstraction for the usage of the previously mentioned component (`ChipSelectField`).
+
+**Example - Basic:**
+
+```tsx
+import { ChipSelect } from '@comet/admin';
+import { MenuItem, ListItemText, ListItemIcon } from '@mui/material';
+import { Offline, Online } from "@comet/admin-icons";
+
+const options = [
+    { label: 'Option 1', value: 'option1' },
+    { label: 'Option 2', value: 'option2' },
+    { label: 'Option 3', value: 'option3' },
+];
+
+const MyComponent = () => {
+    const [someValue, setSomeValue] = React.useState("");
+    
+    return (
+        <ChipSelect options={options} onChange={(e) => setSomeValue(e.target.value)} selectedOption={someValue} />
+    );
+};
+
+
+```
+
+**Example - Final Form:**
+```tsx
+import { ChipSelectField, FinalForm } from '@comet/admin';
+import { MenuItem, ListItemText, ListItemIcon } from '@mui/material';
+import { Offline, Online } from "@comet/admin-icons";
+
+const MyComponent = () => {
```

Comment:
Maybe reduce this example by removing the icons? Otherwise this will be very prominent in the release notes

Reply:
https://github.com/vivid-planet/comet/pull/2186/commits/9780f8b5312bb8133671ac9194932835fd837aca